docs/source/en/api/loaders.mdx

```diff
@@ -35,3 +35,11 @@ Adapters (textual inversion, LoRA, hypernetworks) allow you to modify a diffusio
 ## FromSingleFileMixin
 
 [[autodoc]] loaders.FromSingleFileMixin
+
+## FromOriginalControlnetMixin
+
+[[autodoc]] loaders.FromOriginalControlnetMixin
+
+## FromOriginalVAEMixin
+
+[[autodoc]] loaders. FromOriginalVAEMixin
```

Comment:
Curious, can take it offline too. 

Who came up with the suffix `Mixin`?

Reply:
Transformers haha - the idea here is that `Mixin` means:
- no `__init__(self)`
- has no instance variables, only class variables 

=> So it should be seen really like a "set of tools" added to this class